.gitlab/templates/cached-testrunner.yml

```diff
@@ -5,7 +5,7 @@
     EXT_CACHE_VENV: '${{CI_PROJECT_DIR}}/.cache/ext_cache_venv${{PYTHON_VERSION}}'
   before_script: |
     ulimit -c unlimited
-    pyenv global 3.12 3.8 3.9 3.10 3.11 3.13 3.14.0rc1
+    pyenv global 3.12 3.8 3.9 3.10 3.11 3.13 3.14
```

Comment:
You need to revert this line. This can be updated after we have new testrunner image and update the image hash.